src/plasoscaffolder/bll/services/sqlite_plugin_helper.py

```diff
@@ -1,19 +1,25 @@
 # -*- coding: utf-8 -*-
+# Disable linting until PyCQA/astroid/issues/362 is fixed.
+# pylint: skip-file
 """SQLite plugin helper"""
+import functools
 import os
 import re
 
 from plasoscaffolder.bll.services import base_sqlite_plugin_helper
 from plasoscaffolder.bll.services import base_sqlite_plugin_path_helper
 from plasoscaffolder.dal import base_sql_query_execution
+from plasoscaffolder.model import sql_query_model
 
 
 class SQLitePluginHelper(base_sqlite_plugin_helper.BaseSQLitePluginHelper):
   """Class containing helper functions for the SQLite plugin"""
 
   def __init__(self):
-    """Initializes the sqlite plugin halper"""
+    """Initializes the SQLite plugin helper"""
     super().__init__()
+    self._plugin_name_pattern = re.compile("[a-z]+((_)[a-z]+)*")
```

Comment:
Ok. I changed them to class constants
```
  _PLUGIN_NAME_PATTERN = re.compile("[a-z]+((_)[a-z]+)*")
  _ROW_NAME_PATTERN = re.compile("[A-Z]+([a-zA-Z])*")
```
may i ask why?
May reason was because of the function call re.compile